maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Handlers/SwipeRouteHandlerV2.swift

```diff
@@ -26,7 +26,8 @@ struct SwipeRouteHandlerV2: HTTPHandler {
         let description = "Swipe from \(request.start) to \(request.end) with \(request.duration) duration"
         logger.info("\(description)")
 
-        let eventTarget = EventTarget(bundleId: request.appId)
+        let runningAppId = RunningApp.getForegroundAppId(request.appIds ?? [""])
```

Comment:
Is the empty string needed here?
I'd expect an empty array: `request.appIds ?? []`

Reply:
Done ✅